component_styles/list.css

```diff
@@ -0,0 +1,68 @@
+.list,.nested-list{
+    background-color: white;
+    padding: .5rem 0;
+    margin: 10px;
+    width: 240px;
+    max-width: 95%;
```

Comment:
here instead of this max-width use apply margin, and use max-width for containers with "vw" unit , suggestion, this to will work good.